openapi/openapi.yaml

```diff
@@ -999,6 +976,176 @@ components:
           required:
             - data
 
+    UiCollection:
+      title: 'UiCollection'
+      description: 'Detailed info about a collection used by the ui'
+      type: object
+      properties:
+        all_versions:
+          readOnly: true
+          type: array
+          items:
+            $ref: '#/components/schemas/UiCollectionVersion'
+        community_score:
+          nullable: true
+          type: number
+        community_survey_count:
+          type: integer
+        deprecated:
+          type: boolean
+        download_count:
+          type: integer
+        id:
+          readOnly: true
+          type: integer
+        latest_version:
+          $ref: '#/components/schemas/UiCollectionVersionDetail'
+        name:
+          maxLength: 64
+          type: string
+        namespace:
+          readOnly: true
+          $ref: '#/components/schemas/UiCollectionNamespace'
+        created:
+          format: date-time
+          readOnly: true
+          type: string
+        modified:
+          format: date-time
+          readOnly: true
+          type: string
```

Comment:
created/modified could be removed. @newswangerd does webui use these?

Reply:
We need them to tell when new versions of collections are added.